Dockerfile

```diff
@@ -11,3 +11,8 @@ COPY install_packages.R .
 RUN Rscript install_packages.R
 
 WORKDIR /home/hidden_markov_model
+
+# We'll use this script in our github workflow
+COPY simulation_simple.R .
+COPY hmm_functions.R .
+COPY hmm_parameters.R .
```

Comment:
What happens when the same files exist in the same dir (because of `-v`) when we run `sudo docker run -it -v ~/hidden_markov_model:/home/hidden_markov_model hidden_markov_model bash`?